Microsoft.Toolkit.Services/Microsoft.Toolkit.Services.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
 
   <PropertyGroup>
-    <TargetFrameworks>uap10.0;netstandard1.4;net461</TargetFrameworks>
+    <TargetFrameworks>uap10.0.16299;netstandard1.4;net461</TargetFrameworks>
```

Comment:
Is there a reason to still target netstandard1.4 if 16299 is the min and we have an explicit net* target? 

We should prob move this to `netstandard2.0` across the board as part of this.

Reply:
What's the benefit in moving to a higher .net standard version? I'd imagine if a library can target a lower version it can be used on more platforms.

Reply:
Everything else already supports `netstandard2.0`. UWP was the laggard. There's two big reasons:

1. Simpler dependency graph. NuGet restores are faster, less/no packages are downloaded
2. 50% more types and surface area. More opportunity for cleaner code and refactoring as we go.

Reply:
And the [docs](https://docs.microsoft.com/en-us/dotnet/standard/net-standard#which-net-standard-version-to-target) seem to agree. Updated projects to 2.0